Closes #206 Removed author from programming stratergy. (#212)
pharmaverse/admiraldev: NEWS.md

## Documentation
  - New section in programming strategy regarding comments (#71)

>   - Removed requirement to add `@author` tags to code scripts from programming 
>   strategy, as we will no longer be tracking authors. (#206)

Reviewer: ```suggestion
  - Removed requirement to add `@author` tags to code scripts from programming 
  strategy, as we will only be tracking authors in the DESCRIPTION file. (#206)
```

Author: Ah I see what you meant by your original comment now, thanks.
